Challo/Challo/Persistence/Stores/Booking/BookingModelConvertor.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  BookingModelConvertor.swift
+//  Challo
+//
+//  Created by Kester Ng on 13/4/21.
+//
+
+/*
+class BookingModelConvertor {
```

Comment:
Can remove all these commented out stuff

Reply:
i forgot to remove, okok